pkg/utils/constants.go

```diff
@@ -64,6 +64,12 @@ const (
 	// NodeInstanceIDLabel VPC ID label attached to satellite host
 	NodeInstanceIDLabel = "ibm-cloud.kubernetes.io/vpc-instance-id"
 
+	// MachineTypeLabel is a node label is always seen for instances in satellite cluster
```

Comment:
MachineTypeLabel is a node label always seen for instances in satellite cluster (Remove extra "is")

Reply:
machine-type is not specific to satellite, just mention that label is used to identify the flavor of worker.